### ISSUE TITLE
Institution project (BE)

### ISSUE THREAD

**Issue by MariusJulius** [keeleinstituut/tv-tolkevarav#358]

**Comment by MariusJulius** [comment]
"Asutuse uute tellimuste detailide vaatamine" (view_institution_unclaimed_project_detail) is missing - (tested in #359)

**Comment by kadmit** [comment]
@KaarelKa  the privilege 'VIEW_INSTITUTION_UNCLAIMED_PROJECT_DETAIL` will also show the list of unclaimed projects.

We need to improve FE logic a bit in the next way: In case a user has `VIEW_INSTITUTION_UNCLAIMED_PROJECT_DETAIL` but doesn't have any of `VIEW_INSTITUTION_PROJECT_DETAIL`, `VIEW_INSTITUTION_PROJECT_LIST` then FE should pass filtering by the status with value `NEW` (it means that the project is unclaimed). So the logic itself will be similar to the `VIEW_PERSONAL_PROJECT` logic.

**Comment by KaarelKa** [comment]
@kadmit But I think we can also have unclaimed projects with different statuses (other than NEW). These will not show up then ? For example: https://dev.tolkevarav.eki.ee/projects/9b191627-ff4f-4677-b392-25603b25f45e project

Basically a project can be completed without an assigned PM, someone with the right privileges can just mark things as completed and send it to client, at which point the status of the project will not be NEW anymore.

**Comment by kadmit** [comment]
@KaarelKa Based on the @PlaksoBirgit input I understood that the goal for this privilege is to show projects with the status `NEW` to make them `REGISTERED`.

@PlaksoBirgit could you please correct me if understood it wrong?

**Comment by PlaksoBirgit** [comment]
yes, correct - some institutions might want to have some PM's who only see NEW projects and they can claim and proceed work with these

**Comment by KaarelKa** [comment]
Okay, got it, so other institution projects without a PM/not in the NEW status, should not be visible for users with "VIEW_INSTITUTION_UNCLAIMED_PROJECT_DETAIL" privilege. Those can still be viewed with VIEW_INSTITUTION_PROJECT_DETAIL and VIEW_INSTITUTION_PROJECT_LIST privileges right ?

**Comment by kadmit** [comment]
@KaarelKa Yes, that's correct.

**Comment by KaarelKa** [comment]
Added FE part, can't fully test right now, since we can't pick the VIEW_INSTITUTION_UNCLAIMED_PROJECT_DETAIL privilege, but it should be fine

**Comment by plakitkelly** [comment]
Privilege added

**Comment by plakitkelly** [comment]
@KaarelKa Please check FE part again. User with only view_institution_unclaimed_project_detail sees all projects in Tellimused view (and can open them as well).

**Comment by KaarelKa** [comment]
@plakitkelly Can you add some video or steps to reproduce, ideally also with all the privileges that were picked. Currently tested with minimal privileges (VIEW_PERSONAL_PROJECT + view_institution_unclaimed_project_detail combo) and I could only see projects with status NEW. The "new" filter was checked and couldn't be unchecked.

**Comment by plakitkelly** [comment]
@KaarelKa Now I see only new projects as well.
